src/main/java/de/tum/in/www1/artemis/domain/metis/ConversationParticipant.java

```diff
@@ -29,6 +30,18 @@ public class ConversationParticipant extends DomainObject {
     @NotNull
     private User user;
 
+    /**
+     * Currently only used for {@link de.tum.in.www1.artemis.domain.metis.conversation.Channel}
+     */
+    @Column(name = "is_admin")
+    private Boolean isAdmin;
```

Comment:
See my previous comment, the term "admin" in channels is ambiguous with our definition of a "user admin".

Reply:
> Tested on ts2; unfortunately, I found an issue with an internal server error. ![image](https://user-images.githubusercontent.com/10652693/208123532-e2e41956-b811-42d9-90ae-7f1163e50c23.png) Steps to reproduce: Write message to another user → reply in thread → react with emoji on the answer

I could not reproduce this error, but an existing error causes an internal server error when you too quickly add too many reactions or multiple users do it simultaneously. It was not introduced in this PR. Is it possible that it is this bug? Or does it happen with every reaction, also meaning if you add them slowly? 



https://user-images.githubusercontent.com/29718932/208307169-6e14ec31-1e79-426d-b7bb-971d86a858c1.mp4